docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,449 @@
+
+There are two ways to define these targets:
+
+1. `to.ref`:
+    * **Definition**: Directly references a specific resource.
+    * **Example**: In the `EventPolicy` above, the `my-broker` Broker is directly referenced. This means the `EventPolicy` applies to this specific `Broker`.
+    * **Use Case**: Use `to.ref` when you want to protect a specific resource by name.
+    ```yaml
+    to:
+      - ref:
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+          name: my-broker
+    ```
+2. `to.selector`:
+    * **Definition**: Uses a label selector to match multiple resources of a specific type.
+    * **Example**: The `EventPolicy` includes a `Trigger` with labels matching `app: special-app`. This means the `EventPolicy` applies to all `Triggers` with these labels.
+    * **Use Case**: Use `to.selector` when you want the `EventPolicy` to apply to a group of resources that share common labels.
+
+    ```yaml
+    to:
+      - selector:
+          apiVersion: eventing.knative.dev/v1
+          kind: Trigger
+          matchLabels:
+            app: special-app
+    ```
+
+### Specify who is allowed to send events
+
+The `.spec.from` section specifies **who** is allowed to send events to the targets defined in `.spec.to`. There are two ways to define these sources:
+
+1. `from.ref`:
+    * **Definition**: Directly references a specific event source resource.
+    * **Example**: The `my-source` `PingSource` in another namespace is referenced, meaning this specific source is allowed to send events.
+    * **Use Case**: Use `from.ref` when you want to authorize a specific event source.
+    ```yaml
+    from:
+      - ref:
+          apiVersion: sources.knative.dev/v1
+          kind: PingSource
+          name: my-source
+          namespace: another-namespace
+    ```
+
+2. `from.sub`:
+
+    * **Definition**: Specifies a subject, such as a service account, that is allowed to send events. It can include wildcard patterns as a postfix (`*`) for broader matching.
+    * **Example**: The `EventPolicy` allows events from the `trusted-app` service account in the default namespace and any service account in `default` namespace that starts with `other-`.
+    * **Use Case**: Use `from.sub` to allow specific users or service accounts, or to apply wildcard patterns for more flexibility.
+    ```yaml
+    from:
+      - sub: system:serviceaccount:default:trusted-app
+      - sub: "system:serviceaccount:default:other-*"
+    ```
+
+### Advanced CloudEvent filtering criterias
+
+The `.spec.filters` section is optional and specifies additional criteria that the event itself must meet to be allowed. If the filters are specified, an event must match at least one of them to be accepted. `.spec.filters` accepts the same filter dialects as [Triggers](../triggers/README.md#supported-filter-dialects).
```

Comment:
let's show an example with filters? 

I'd also describe what happens with multiple policies combined with filters

Reply:
In line 59 we have an example with filters. Or do you mean in the later "bigger" example?

Reply:
> I'd also describe what happens with multiple policies combined with filters
Can you describe what you mean? Shouldn't it be the same on multiple policies with or without filters? As long as one policy allows the request (having filters or not), the request is passed

Reply:
Updated